packages/cadl-ranch-specs/http/content-type/multipart-formdata/main.tsp

```diff
@@ -0,0 +1,74 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Test content-type: multipart/form-data")
+@scenarioService("/content-type/multipart-formdata")
+namespace ContentType.MultipartFormData;
```

Comment:
```suggestion
namespace Payload.MultiPart
```
or `Payload.MultiPart.FormData` if we see some different handling in the future for other types of multipart


to be in the same folder as the others